frontend/ui/widget/inputtext.lua

```diff
@@ -237,11 +237,10 @@ if Device:isTouchDevice() or Device:hasDPad() then
                             },
                             {
                                 text = _("Paste"),
+                                enabled = not is_clipboard_empty,
                                 callback = function()
-                                    if not is_clipboard_empty then
-                                        UIManager:close(clipboard_dialog)
-                                        self:addChars(clipboard_value)
-                                    end
+                                    UIManager:close(clipboard_dialog)
+                                    self:addChars(clipboard_value)
```

Comment:
Can't the clipboard become non-empty while in there, if you Copy line, and then want to hit Paste ?

Reply:
The dialog get closed after pressing any "Copy ..." button.